.github/workflows/trigger-all.yml

```diff
@@ -32,8 +32,8 @@ jobs:
             wf_category: ${{ inputs.wf_category || 'NIGHTLY' }}
             gitref: ${{ inputs.gitref || 'main' }}
             push_to_pypi: ${{ (github.event.schedule == '30 0 * * *') || inputs.push_to_pypi || false }}
-            test_configs: '[{"python":"3.11.4","label":"ubuntu-24.04","timeout":"40","code_coverage":true},
-                            {"python":"3.10.12","label":"ubuntu-22.04","timeout":"40"},
+            test_configs: '[{"python":"3.11.4","label":"k8s-util","timeout":"40","code_coverage":true},
```

Comment:
If we switch to k8s-util we only need one job here now as the original ones were to cover different ubuntu versions.

Reply:
ok sounds good.